src/Simulation/QSharpFoundation/Diagnostics/Assert.qs

```diff
@@ -51,6 +52,8 @@ namespace Microsoft.Quantum.Diagnostics {
     /// The probability with which the given result is expected.
     /// ## msg
     /// A message to be reported if the assertion fails.
+    /// ## tolerance
+    /// Tolerance of the measurement probability.
```

Comment:
```suggestion
    /// Tolerance allowed in the measurement probability.
```

Reply:
The PR was scheduled to auto-merge. After you approved it got merged in automatically.
I will try to make this change in the subsequent edits, other PRs.